selenium_tests/pages.py

```diff
@@ -21,16 +23,97 @@ def navbar(self) -> 'NavbarElement':
         return NavbarElement(self.driver)
 
 
+class LoginPage(BasePage):
+
+
+    def __init__(self, driver: WebDriver) -> None:
+        self.driver = driver
+
+        self.login_user_name = driver.find_element(By.NAME, Locator.login_user_name)
+        self.login_password = driver.find_element(By.NAME, Locator.login_password)
+        self.login_submit = driver.find_element(By.XPATH, Locator.login_submit)
+
+    def get_username(self):
+        return self.login_user_name
+ 
+    def get_password(self):
+        return self.login_password
+ 
+    def get_login_submit(self):
+        return self.login_submit
+
+    def login_admin_user(self):
+        self.user = AdminUserFactory()
+        self.login_user_name.send_keys(self.user.username)
+        self.login_password.send_keys('test@123')
+        self.login_submit.click()
+
+    def login_user(self):
+        self.user = UserFactory()
+        self.login_user_name.send_keys(self.user.username)
+        self.login_password.send_keys('test@123')
+        self.login_submit.click()
+
+class HomePage(BasePage):
+
+    def __init__(self, driver: WebDriver) -> None:
+        self.driver = driver
+        self.login_button = driver.find_element(By.XPATH, Locator.login_button)
+ 
+    def get_login(self):
+        return self.login_button
+
+class ArticlePage(BasePage):
+
+    def __init__(self, driver: WebDriver) -> None:
+        self.driver = driver
+
+        self.article_title = driver.find_element(By.TAG_NAME,Locator.article_heading)
```

Comment:
I would extract this to a separate method, similar to `FooterItemElement.title`:

```
@property
def title(self) -> str:
    return driver.find_element(By.TAG_NAME, 'h1').text
```